boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/iam/util/JWTUtils.java

```diff
@@ -40,6 +40,7 @@
 import org.jose4j.jwt.consumer.JwtConsumerBuilder;
 import org.jose4j.keys.AesKey;
 import org.jose4j.lang.JoseException;
+import org.ow2.proactive.boot.microservices.iam.exceptions.IAMException;
```

Comment:
I don't see any other change using this import

Reply:
The name of the package changed.